lms/templates/shoppingcart/receipt.html

```diff
@@ -108,73 +108,73 @@ <h3 class="text-center">${_("Please send each professional one of these unique r
               <div class="row-inside">
                 <p>
                   <b>${_('Company Name')}:</b>
-                  <label>
+                  <span>
```

Comment:
could be great to add a class to this for styling purposes: .billing-detail-value perhaps?  (the <b> tag above it could also be .billing-detail-label). After this change, the styles should be set through a class instead of the <span> tag.